website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -436,6 +436,8 @@ A `default_node_pool` block supports the following:
 
 * `scale_down_mode` - (Optional) Specifies the autoscaling behaviour of the Kubernetes Cluster. Allowed values are `Delete` and `Deallocate`. Defaults to `Delete`.
 
+* `snapshot_id` - (Optional) The ID of the Snapshot which should be used to create this default Node Pool.
```

Comment:
Does it support a rolling update? Or will the node pool be recreated on snapshot_id change?

Reply:
Hi @js-315385995 , yes, it could be updated, but it is done by cycling the system node pool of the cluster, `temporary_name_for_rotation` must be specified when changing it. I've updated the doc as well.